lib/matplotlib/animation.py

```diff
@@ -576,6 +575,45 @@ def cleanup(self):
                 os.remove(fname)
 
 
+@writers.register('pillow')
+class PillowWriter(MovieWriter):
+    @classmethod
+    def isAvailable(cls):
```

Comment:
Is there any reason this is `cls` and not `self`? (just asking out of curiosity!)

Reply:
because you need to check whether the class is available before instantiating one (at least that's how the animation framework is setup).